hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/exceptions/OMException.java

```diff
@@ -259,6 +259,8 @@ public enum ResultCodes {
 
     FEATURE_NOT_ENABLED,
 
-    INVALID_SNAPSHOT_ERROR
+    INVALID_SNAPSHOT_ERROR,
+
+    SNAPSHOT_EXISTS,
```

Comment:
Don't know if it is just me or `SNAPSHOT_EXISTS` sounds like "snapshot already exists". Would it be better if we use `CONTAINS_SNAPSHOT`. Or something along the line which tells customer that bucket or volume can't be deleted because they contains snapshot.